api/db/populate-users.js

```diff
@@ -140,10 +140,9 @@ const createUsers = async (multiplier, randomLocation, verbose) => {
     usersArray.push(await createUser(userParams));
   }
 
-  console.log('\x1b[32m', `Created ${usersArray.length} regular users`);
   console.log(
     '\x1b[32m',
-    `Created ${notificationsCount} notifications related to creation of a new user`
+    `Created ${usersArray.length} users (and ${notificationsCount} related notifications)`
```

Comment:
Did we agree to count the admin here too in the end? This can be easily done by pushing it too to the users' array. And while we are at it, the function that creates the admin user is missing the `async` and `await` keywords